tests/test_harvesters.py

```diff
@@ -191,33 +191,35 @@ def test_creodias_harvester(self):
         self.assertIsInstance(harvester._current_crawler, crawlers.CreodiasEOFinderCrawler)
         self.assertIsInstance(harvester._ingester, ingesters.CreodiasEOFinderIngester)
 
-    def test_osisaf_harvester_extra_excludes(self):
-        """ extra excludes should have passed by the excludes as a list in configuration file.
+    def test_osisaf_harvester_include(self):
+        """ include criteria should have passed by the "includes" as a list in configuration file.
```

Comment:
Please update the complete docstring, don't just change the name (in the whole PR, not just here).
This parameter is no longer a list.

Reply:
SOrr.thanks.updated.